shared/charts/backup-exporter/templates/configmap.yaml

```diff
@@ -0,0 +1,22 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "backup-exporter.name" . }}
+  namespace: backup-exporter
+  labels:
+    app: {{ template "backup-exporter.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  backup.sh: |
+    set -e
+    gcloud config set account {{ .Values.serviceAccountName }}
+    SNAPSHOTS=$(for i in `seq 0 $(({{ .Values.replicaCount }}-1))`; do gcloud compute snapshots list --sort-by=~creationTimestamp,STATUS --limit=1 --format="value[separator=';'](name,status)" --filter="name~'pv-database-storage-couchdb-couchdb-$i-*'" | cut -f1 -d\; ; done)
+    DESTINATION_BUCKET={{ .Values.destinationBucket }}
+    TIMESTAMP=$(date +%s)
```

Comment:
Can we use a human-readable timestamp format?

Reply:
Moved this task to the description of the next PR. https://github.com/gpii-ops/gpii-infra/pull/349